### ISSUE TITLE
Feature Request: Success messages

### ISSUE THREAD

**Issue by tobiasgruber1995** [filegator/filegator#261]
As of now when performing an operation like zipping/unzipping only in case of an error you get any "response". In case of success you only see the file(s) appear, but sometimes still an error occurred (e.g. not everything was zipped/unzipped) and you don't realize right away. Since you need to wait anyways for the operation to finish ("Waiting circle" keeps turning until everything is done, because no asynchrony is used - see [Issue #240](https://github.com/filegator/filegator/issues/240)), it would be nice to get some kind of "success message" once everything actually finished successfully.

**Comment by Digioso** [comment]
I'm using version 7.9.3 .
I'd like to add to this request: Please give error messages as well.
For example if you want to create a folder in a directory where you do not have write access, the folder is not created (makes sense, duh :) ).
But unfortunately there is no popup or anything telling you that there was an error.

**Comment by vdbhb59** [comment]
I agree. I landed from [HestiaCP](https://github.com/hestiacp/hestiacp/issues/4176) here.
Turns out, all it gives is an "unknown error" while in reality nothing happens. Deletion does not work, and especially if there are multiple subfiles & subfolders, nothing happens at all. Within 2 seconds it does a roundabout and then just displays "unknown error", or at times displays nothing at all.

Finally, if say I have deleted some files and folder either via filegator, or even via some FTP session, those files remain on filegator, even though they are deleted in reality.

**Comment by virtualos** [comment]
> I agree. I landed from [HestiaCP](https://github.com/hestiacp/hestiacp/issues/4176) here. Turns out, all it gives is an "unknown error" while in reality nothing happens. Deletion does not work, and especially if there are multiple subfiles & subfolders, nothing happens at all. Within 2 seconds it does a roundabout and then just displays "unknown error", or at times displays nothing at all.
> 
> Finally, if say I have deleted some files and folder either via filegator, or even via some FTP session, those files remain on filegator, even though they are deleted in reality.

Me too
